data_validation/combiner.py

```diff
@@ -34,27 +36,106 @@
     from data_validation.metadata import RunMetadata, ValidationMetadata
 
 
+# At around 140 columns we hit RecursionError when unioning Ibis subqueries.
+# This constant is a threshold at which we slice up the input Dataframes
+# and stitch them back together again after Ibis processing.
+COMBINER_COLUMN_SLICE_WIDTH = 120
+
 COMBINER_GET_SUMMARY_EXC_TEXT = (
     "Error while generating summary report of row validation results"
 )
 
 
 def generate_report(
```

Comment:
```suggestion
def get_validation_report(
```
I really liked the approach you took on having this wrapper around the main logic. But I think we should create specific names for each of them to let it very clear what which one is intended to be used for, instead of relying only on function documentation.

Reply:
I left this name the same as before I started making changes but did rename the sub-function.

Is that any better or would still recommend renaming the top level function?

Reply:
Yep, changing only the sub-function name already makes a difference to distinguish them, seems great to me! I'll resolve this comment.